jamjar/jamjar/videos/models.py

```diff
@@ -11,10 +11,12 @@
 class Video(BaseModel):
 
     name = models.CharField(max_length=128)
+    length = models.FloatField(null=True, blank=True)
```

Comment:
Although I don't image this is gonna be an issue in this instance, a good thing to note - https://docs.djangoproject.com/es/1.9/ref/models/fields/#floatfield-vs-decimalfield